connector/src/main/java/org/geysermc/connector/GeyserConnector.java

```diff
@@ -325,6 +322,38 @@ public void removePlayer(GeyserSession player) {
         players.remove(player);
     }
 
+    /**
+     * Gets a player by their current UUID
+     *
+     * @param uuid the uuid
+     * @return the player or <code>null</code> if there is no player online with this UUID
+     */
+    public GeyserSession getPlayerByUuid(UUID uuid) {
+        for (GeyserSession session : players) {
+            if (session.getPlayerEntity().getUuid().equals(uuid)) {
+                return session;
+            }
+        }
+
+        return null;
+    }
+
+    /**
+     * Gets a player by their Xbox UUID
+     *
+     * @param xboxUuid the Xbox UUID
+     * @return the player or <code>null</code> if there is no player online with this XUID
+     */
+    public GeyserSession getPlayerByXboxUuid(String xboxUuid) {
```

Comment:
Name it `getPlayerByXuid`

Reply:
**U**niversally **U**nique **ID**dentifier
**X**box **U**ser **ID**entifier (or something)

Reply:
I named it xboxuuid because it's named like this in AuthData

Reply:
It shouldn't be

Reply:
If you could make this change, then this PR is good to go.